changelog/1.20.0.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.20.0"
+description: "Released on 06/16/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.20.0.
+
+### Features ✨
+
+- web: A site-wide workspace template policy may now be managed at **Manage >
+  Admin > Templates > Template Policy**. If not set, a sensible default is
+  provided.
+- web: The _nodeSelectors_ field was added to workspace templates
+- other: There is a new JsonSchema for writing Coder workspaces as code template
+  with intellisense
+- web: Added a service message that's displayed to all users of the system. The
+  message may be dismissed by each user at any point and is not shown again
+  until there is a new message.
+- web: System banners can now wrap text
+- infra: TLS management (/_TODO_/)
+- web: Opt-in for HTTP Strict Transport Security was added. This setting can be
+  managed at **Manage > Admin > Infrastructure > HTTP Strict Transport
+  Security**
+- web: Opt-in for Secure Cookies was added. This setting can be managed at
+  **Manage > Admin > Infrastructure > Secure Cookie**
+- templates: Kubernetes annotations are now available in workspace templates
+- infra: Increased network performance for creating new workspaces
+- infra: A `CODER_RUNTIME` environment variable was added
+- web: Decomissioned workspaces (awaiting deletion) are now visible in the UI
+
+### Bug fixes 🐛
+
+- web: Fixed a bug that caused duplicate fetch requests on page load
+- web: Fixed an issue that caused a private dev URL to load as a blank page for
+  other users. Other usess now see a meaningful error
+
+### Security updates 🔐
+
+- web: Previously, information about other users was visible in payloads for
+  workspace images. This information has been remove; users can now only see
+  their own data
```

Comment:
CC @Emyrk to verify

Reply:
Correct. All all **Images** tabs, every user workspace was listed (by image and by image tag). Now this page only shows the user's workspaces (admins can still see all)